Gemfile

```diff
@@ -1,2 +1,6 @@
 source "https://rubygems.org"
 gemspec :name => 'uswds-jekyll'
+
+group :jekyll_plugins do
+  gem 'jekyll_pages_api_search'
```

Comment:
Adding the gem

Reply:
You will need to tell users to do this in the README. Adding it to the `gemspec`(s?) is better, but that won't automatically add it to the `jekyll_plugins` `group`.